sync/storage/batch/batch.go

```diff
@@ -94,7 +109,7 @@ func (s *batchStorageSync) syncBucket(ctx context.Context, lastSuccessfulRun tim
 	for _, f := range files {
 		if time.Time(f.Created).After(lastSuccessfulRun) {
 			syncCount++
-			go s.syncFile(ctx, lastSuccessfulRun, bucketID, f.Name, ch)
+			go s.syncFile(ctx, lastSuccessfulRun, bucketID, f.Name, ch, fileRateLimit)
```

Comment:
invalid operation: f (variable of type *invalid type) has no field or method Name